### PR TITLE
Remove three unused tools

### PR DESCRIPTION
**Pull request by optimumtact**
These tools are unused as far as I am aware.

If people want to keep source code around then we can open a new repo for dead code on the tgstation org

### CODE REVIEW

File: tools/midi2piano/midi2piano.sln [hunk 1]
```diff
@@ -1,30 +0,0 @@
-﻿
-Microsoft Visual Studio Solution File, Format Version 11.00
```
Comment (Iamgoofball) [inline tools/midi2piano/midi2piano.sln]: People still use midi2piano

File: tools/UnstandardnessTestForDM/UnstandardnessTestForDM/Form1.cs [hunk 2]
```diff
@@ -1,484 +0,0 @@
-﻿using System;
```
Comment (Iamgoofball) [inline tools/UnstandardnessTestForDM/UnstandardnessTestForDM/Form1.cs]: Can we keep the line checker at least? This is nice for running every once in awhile and seeing how many lines of code we have.

Comment (Jordie0608) [inline tools/UnstandardnessTestForDM/UnstandardnessTestForDM/Form1.cs]: DM Line Counter.exe does this already

Comment (Iamgoofball) [inline tools/UnstandardnessTestForDM/UnstandardnessTestForDM/Form1.cs]: i thought this was the DM line counter

**Comment by Dorsisdwarf** [comment]
Ready4Merge?

**Comment by optimumtact** [comment]
No gotta readd midi2piano

**Comment by Razharas** [comment]
@optimumtact

**Comment by optimumtact** [comment]
@Razharas ready to go assuming travis passes

**Pull request merged**